cg/meta/archive/models.py

```diff
@@ -1,34 +1,33 @@
 """Contains base models to be inherited from in other archive software files."""
-from abc import abstractmethod, ABC
+from abc import abstractmethod
 from typing import List
 
+from cg.store.models import Sample
 from housekeeper.store.models import File
 from pydantic import BaseModel
 
-from cg.store.models import Sample
-
 
-class ArchiveFile(ABC, BaseModel):
+class FileTransferData(BaseModel):
     """Base class for classes representing files to be archived."""
 
     @classmethod
     @abstractmethod
     def from_file_and_sample(cls, file: File, sample: Sample) -> "ArchiveFile":
         """Instantiates the class from a File and Sample object."""
-        raise NotImplementedError
+        pass
 
 
-class ArchiveInterface(ABC):
+class ArchiveHandler:
```

Comment:
A bit uncertain why this was changed from ArchiveInterface to ArchiveHandler by VJ. Is there a consensus that Handler should be used here? I am fine with it either way.

Reply:
in cg we use handlers. You could have a look at those and see if they do similar things but with different context. If that is the case we could argue for using Handler here as well for consistency